1.9/networking/load-balancing-vips/virtual-ip-addresses.md

```diff
@@ -21,81 +23,75 @@ VIPs follow this naming convention:
 <service-name>.marathon.l4lb.thisdcos.directory:<port>
 ```
 
-## Prerequisite:
+### Prerequisite
 
 *   A pool of VIP addresses that are unique to your application.
 
-## Create a VIP:
+## Creating a VIP
 
-1.  From the DC/OS [web interface](/docs/1.9/gui/), click on the **Services** tab and either click your service name or click **RUN A SERVICE** to create a new service.
+1.  From the DC/OS [GUI](/docs/1.9/gui/), click on the **Services** then **RUN A SERVICE**.
```

Comment:
I assume you mean this 
![image](https://cloud.githubusercontent.com/assets/11722533/26748137/f92e4f8a-47af-11e7-8fc4-df534a63f384.png)

not this:
![image](https://cloud.githubusercontent.com/assets/11722533/26748117/c6043da4-47af-11e7-853d-d48e0e1ea8fb.png)